src/controllers/commands/CommandController.cpp

```diff
@@ -518,7 +554,15 @@ QString CommandController::execCustomCommand(const QStringList &words,
         result = result.mid(1);
     }
 
-    return result.replace("{{", "{");
+    auto res = result.replace("{{", "{");
+
+    if (dryRun)
+    {
+        return res;
+    }
+
+    return appendWhisperMessageLocally(res) ? (sendWhisperMessage(res), "")

```

Comment:
Expand this to a logical if-statement instead of using a ternary operator. also what does `(sendWhisperMessage(res), "")` do? does that execute sendWhisperMessage and return an empty string?

Reply:
yes, a comma operator
sure

Reply:
https://github.com/fourtf/chatterino2/blob/b7d04a648ff7ab584c07f5d9d58defe89b393253/src/controllers/commands/CommandController.cpp#L563-L571